src/mapper/mapper.go

```diff
@@ -79,52 +79,50 @@ func match(dk *dynatracev1beta1.DynaKube, namespace *corev1.Namespace) (bool, er
 // updateNamespace tries to match the namespace to every dynakube with codeModules
 // finds conflicting dynakubes(2 dynakube with codeModules on the same namespace)
 // adds/updates/removes labels from the namespace.
-func updateNamespace(namespace *corev1.Namespace, dkList *dynatracev1beta1.DynaKubeList) (bool, error) {
-	var updated bool
+func updateNamespace(namespace *corev1.Namespace, installedDynaKubes *dynatracev1beta1.DynaKubeList) (bool, error) {
+	namespaceUpdated := false
```

Comment:
```suggestion
	updated := false
```
nitpick:
we are in updateNamespace, we don't have to be so verbose (also you didn't update the name here: `updateLabels` 😅 )

Reply:
That's to clearly distinguish between the two update flags (`updated` and `upd`) uses here and increase readability. 'updateLabels' only uses one flag.